src/sass/components/Pagination.scss

```diff
@@ -26,7 +26,16 @@
   opacity: 1;
   transition: opacity 0.3s;
 
+  td {
+    opacity: 1;
+    transition: opacity 0.3s;
+  }
   &.fadeOut {
-    opacity: 0;
+    opacity: 0.5;
+    box-shadow: 0 1px 1px -2px;
```

Comment:
Without the box-shadow, the bottom border thins by ~1px on fade in a somewhat distracting manner. By adding it in, it APPEARS to do nothing, which is exactly what we want ^^;